src/components/DisplayAccounts.jsx

```diff
@@ -49,14 +49,15 @@ export default function DisplayAccounts({ accounts, ...props }) {
                       <code className="text-break">{xpub}</code>
                     </rb.Col>
                   </rb.Row>
-                  {entries.map(({ address, amount, hd_path: hdPath, labels }) => (
+                  {entries.map(({ address, amount, hd_path: hdPath, labels, status }) => (
                     <rb.Row key={address} className="w-100 mt-3">
                       <rb.Col xs="auto">
                         <code className="text-break">{hdPath}</code>
                       </rb.Col>
                       <rb.Col>
                         <code className="text-break">{address}</code>{' '}
                         {labels && <span className="badge bg-info">{labels}</span>}
+                        {status && <span className="badge bg-info">{status}</span>}
```

Comment:
The `satus` of an address is if it is _new_ or _used_. This must've been the `label` in older API versions. I'm pretty sure I saw that info in the UI before. The label seems to be empty now. I added the status to the UI to make sure it's shown in any case.